model-optimizer/CMakeLists.txt

```diff
@@ -54,3 +54,9 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/automation
         DESTINATION deployment_tools/model_optimizer
         COMPONENT tests
         EXCLUDE_FROM_ALL)
+
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
+        DESTINATION deployment_tools/model_optimizer
```

Comment:
Deployment tools is obsolete folder. Tests files should be installed somewhere in tests